src/client/swagger/notebooks.yml

```diff
@@ -57,6 +57,11 @@ paths:
   /notebooks:
     get:
       description: get all records
+      parameters:
+        - name: orgID
+          in: query
+          type: string
+          required: true
```

Comment:
This file is actually generated so we'll need to make the change to notebooksd over in IDPE. I'm currently working on a massive refactor to notebooksd and I could include this in the refactor or in a PR right after the refactor is complete https://github.com/influxdata/idpe/pull/10606

Reply:
Ahhh, I didn't realize that. Whatever's easiest for you as far as when it include it is fine with me. It's not really blocking for me on OSS right now since I can just use this branch for the frontend and get the interaction I need.

Reply:
Ok! I should have this refactor done today or tomorrow hopefully. So we should be able to get the change in by the end of the week. I'll let you know

And I don't know if I'll keep the swagger generation in notebooksd forever...but for now it makes it easy for me to evolve the api so it's useful

Reply:
added [here](https://github.com/influxdata/idpe/pull/10606/commits/23e31f26f826c282b53d12e718f4d4941e3352ad) for [this PR](https://github.com/influxdata/idpe/pull/10606)